cypress/integration/it_can_log_user_in_and_out.js

```diff
@@ -0,0 +1,20 @@
+describe('Languages Supported Page', () => {
+  it('can login user', () => {
+    cy.visit('/')
+
+    cy.contains('login').click()
+
+    cy.contains('Logged in as: LoganRools')
+    cy.contains('logout')
+  })
+
+  it('can logout user', () => {
+    cy.visit('/')
+
+    cy.contains('login').click()
+    cy.contains('logout').click()
+
+    cy.contains('Logged in as: LoganRools').should('not.exist')
+    cy.contains('login')
+  })
+})
```

Comment:
Just tests that username is displayed but doesn't actually test user context. Will add more robust test once this actual functionality is implemented.

Reply:
This seems fine for now.